CMakeLists.txt

```diff
@@ -174,7 +174,7 @@ target_link_options(Edyn
 )
 
 if(MSVC)
-    target_compile_options(Edyn PRIVATE /W4)
+    target_compile_options(Edyn PRIVATE /W4 /bigobj)
```

Comment:
I don't fully understand the involved tooling, but debug builds were failing to produce the library file without this. I don't know if it's related to the entt upgrade - happy to remove it from the pull request and keep it in my own fork.

Reply:
I am not sure what that is. I will have to test it and see.

Reply:
```
Most modules never generate an .obj file that contains more than 65,279 sections. However, machine-generated code, or code that makes heavy use of template libraries, may require .obj files that can hold more sections. /bigobj is enabled by default on Universal Windows Platform (UWP) projects because the machine-generated XAML code includes a large number of headers. If you disable this option on a UWP app project, your code may generate compiler error C1128.

For information on the PE-COFF object file format, see [PE Format](https://docs.microsoft.com/en-us/windows/win32/debug/pe-format) in the Windows documentation.
```
https://docs.microsoft.com/en-us/cpp/build/reference/bigobj-increase-number-of-sections-in-dot-obj-file?view=msvc-170

should be save to use for all msvc build. c++ templates generate A LOT of debug info...